astrocut/cutouts.py

```diff
@@ -594,6 +594,11 @@ def img_cut(input_files, coordinates, cutout_size, stretch='asinh', minmax_perce
         cutout_path = []
         for fle in input_files:
 
+            if cutout_hdu_dict.get(fle) is None:
```

Comment:
i think this check is fine, but seems like there might be something going wrong here: https://github.com/spacetelescope/astrocut/blob/56168752cfcd49ff948ab3443f11dbf644246a06/astrocut/cutouts.py#L554

it's just not clear to me how this could be `None`

is there a check that we should but in there as well?

is it possible to add a test that failed before and will now pass with this additional check?

Reply:
Ah, that does change the game if so.  I'm meeting with @phicc tomorrow to talk through further, stay tuned!  Just so I know, this would be z.MAST only that would be impacted by Astrocut changes that relates to JWST?  No Portal, etc. reliance?

Reply:
This is wrapped in an if, so it seems like this could be None if `cutout` is all zeros. It seems like setting `cutout_hdu_dict` to `default_dict(list)` would maybe fix this issue and would be consistent with `cutout_hdu_dict.get(in_fle, [])`.

https://github.com/spacetelescope/astrocut/blob/56168752cfcd49ff948ab3443f11dbf644246a06/astrocut/cutouts.py#L553-L554